src/resource/file.ts

```diff
@@ -19,9 +19,9 @@
 // SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
 
-import type { Buffer, File as NodeFile } from "buffer";
+import type { File as NodeFile } from "buffer";
 import type {
-  File,
+  File as BlobFile,
```

Comment:
We have to remove that alias eventually, it's really confusing ^^. When Node 18 goes away, that's going with it.